.github/workflows/lint.yml

```
@@ -49,10 +39,10 @@ jobs:
               STATUS=1
             fi
           }
-
-          assert-nothing-changed go fmt ./...
           assert-nothing-changed go mod tidy
-
-          bin/golangci-lint run --out-format=colored-line-number --timeout=3m || STATUS=$?
-
           exit $STATUS
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@4afd733a84b1f43292c63897423277bb7f4313a9
```

If we put a comment like this, Dependabot will bump the comment too when it updates this in the future.

```suggestion
        uses: golangci/golangci-lint-action@4afd733a84b1f43292c63897423277bb7f4313a9 # v8.0.0
```

---

Out of all the comments, I think this might be the only hard **suggest** I'd consider.

---

I had no idea that the comment had that behaviour. Great suggestion.